eng/pipelines/templates/jobs/ci.yml

```diff
@@ -78,12 +78,18 @@ jobs:
           echo "##vso[build.addbuildtag]Scheduled"
         displayName: "Tag scheduled builds"
         condition: and(eq(variables['Build.SourceBranchName'],variables['DefaultBranch']),eq(variables['Build.Reason'],'Schedule'))
-      # Both 'Dump property properties' and 'Update package properties with dev version' serve the same purpose of generating package json.
-      # We decided to only run the later one (update properties) in all pipelines for the following reasons:
-      # 1. 'Update properties' happens after 'build', we will have more information at the 'update properties' step.
-      # 2. We did not find any usage of package json before 'update properties' in .NET, so it is safe to skip the 'dump properties' for now.
+      # Set env variable SkipDevBuildNumber to generate package properties with non alpha version
+      # Package version is used to create or update package work item from CI pipeline
+      - pwsh: |
+          echo "##vso[task.setvariable variable=SkipDevBuildNumber]true"
+        displayName: "Set SkipDevBuildNumber env variable"
       - template: /eng/common/pipelines/templates/steps/daily-dev-build-variable.yml
+        parameters:
+          ServiceDirectory: ${{parameters.ServiceDirectory}}
       - pwsh: |
+          # Reset SkipDevBuildNumber as empty
+          echo "##vso[task.setvariable variable=SkipDevBuildNumber]"
+          Write-Host "SkipDevBuildNumber: [$(SkipDevBuildNumber)]"
```

Comment:
This write-host will not have the updated value. It will have the value at the beginning of this step.

Reply:
Correct. That's what confused me when I looked at the logs.

Reply:
I would suggest still remove this logging as it will just confuse things.